specs/HOOK-hooks/README.md

```diff
@@ -477,3 +477,32 @@ An SDK may use any optional type, but for a JSON representation `null` should be
 ##### Explanation
 
 Replace is false, so this is an update of "flagB" and "flagC". "flagB" has been deleted, and "flagC" has been created or updated.
+
+## 1.6 Track Series
+
+Hook stages that are executed during the execution of a track method call.
+
+### Requirement 1.6.1
+
+> Hooks **MUST** support a `afterTrack` handler. It accepts an `TrackSeriesContext` and has no return value.
+
+### Requirement 1.6.2
+
+> The `afterTrack` handler **MUST** be executed during the execution of the track method after the `custom` event has been enqueued.
+
+For track there isn't a meaningful before/after as tracking is a point-in-time operation, so only the `afterTrack` is being specified at this time.
+
+If mutability is added to hooks in the future, then `beforeTrack` could be added, and the signature extended with a `hook data` parameter and return.
+
+If the no `custom` event could be enqueued, because of an invalid context or JSON data, then the `afterTrack` method will not be called.
```

Comment:
```suggestion
If no `custom` event could be enqueued, because of an invalid context or JSON data, then the `afterTrack` method will not be called.
```